test/cookbooks/docker_test/recipes/container.rb

```diff
@@ -1213,6 +1213,7 @@
   memory_swap '5M'
   memory_swappiness 50
   memory_reservation '5m'
+  shm_size '32m'
```

Comment:
can you complete this test by adding the matching inspec line in `test/integration/resources/inspec/assert_functioning_spec.rb`, bottom of the file is the memory container.